pkg/cmd/pr/create/create.go

```diff
@@ -440,7 +440,8 @@ func createRun(opts *CreateOptions) error {
 		if err != nil {
 			return err
 		}
-		return submitPR(*opts, *ctx, *state)
+		// TODO wm: revisit project support
```

Comment:
Anywhere you see this comment is because I'm assuming that projects v1 is supported for backwards compat reasons as I pass through `gh.ProjectsV1Supported` to functions used by both `issue create` (changed here) and `pr create`).